### PR TITLE
Add missing packet kind for bootstrap packet.

### PR DESCRIPTION
**Pull request by iphydf**
This change is [Reviewable](https://reviewable.io/reviews/toktok/hstox/66)

### CODE REVIEW

**Comment by Diadlo** [comment]
Reviewed 2 of 3 files at r1.
Review status: 2 of 3 files reviewed at latest revision, 1 unresolved discussion.

---

*[test/toxcore/packet_kinds.c, line 46 \[r1\]](https://reviewable.io:443/reviews/toktok/hstox/66#-KQ_TtFAWX8h4ylxnsKT:-KQ_TtFAWX8h4ylxnsKU:b-76214u) ([raw file](https://github.com/toktok/hstox/blob/e3ebe813224de2ae4cc6700f16aecc59d17ac662/test/toxcore/packet_kinds.c#L46)):*
> ```C
>     // | OnionResponse1    -- 0x8e: First level wrapped onion response
>     NET_PACKET_ONION_RECV_1,
>     // TODO(iphydf): We don't know what this is, yet.
> ```

Remove TODO?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/hstox/66)*

**Comment by iphydf** [comment]
Review status: 2 of 3 files reviewed at latest revision, 1 unresolved discussion.

---

*[test/toxcore/packet_kinds.c, line 46 \[r1\]](https://reviewable.io:443/reviews/toktok/hstox/66#-KQ_TtFAWX8h4ylxnsKT:-KQ_d7X0_BsqBhigxQwN:ba7x1vv) ([raw file](https://github.com/toktok/hstox/blob/e3ebe813224de2ae4cc6700f16aecc59d17ac662/test/toxcore/packet_kinds.c#L46)):*
<details><summary><i>Previously, Diadlo (Diadlo) wrote…</i></summary>
> Remove TODO?
</details>
We still don't know what it is.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/hstox/66)*

**Comment by GrayHatter** [comment]
:lgtm_strong:

---

Reviewed 3 of 3 files at r1, 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/hstox/66#-:-KQvw7uOZnXUNT_NMuhC:b-jbsf8z)*

File: src/tox/Network/Tox/DHT.lhs [hunk 1]
```diff
@@ -271,3 +271,18 @@ sending ping packets to his one IP/port.  B will detect that A has a symmetric
...
+additional request kind: Bootstrap Info. The request is simply a packet of
+length 78 bytes where the first byte is 0xf0. The other bytes are ignored.
+
+The response format is as follows:
+
+\begin{tabular}{l|l|l}
+  Length             & Type        & \href{#protocol-packet}{Contents} \\
+  \hline
+  \texttt{4}         & Word32      & Bootstrap node version \\
+  \texttt{73}        & Bytes       & Message of the day \\
```
Comment (nurupo) [inline src/tox/Network/Tox/DHT.lhs]: 73? The response is not limited to the 78 bytes. https://github.com/TokTok/toxcore/blob/master/other/bootstrap_node_packets.h#L31

**Pull request closed without merging**